main.py

```diff
@@ -0,0 +1,32 @@
+# step 1
+def tokenize(text: str) -> list[str] | None:
```

Comment:
I do not think that you should implement functions here! please move it to lab_1../main.py